scripts/e2e.sh

```diff
@@ -69,7 +72,11 @@ cat fabric-operator/values.yaml | sed "s/<replaced-ingress-nginx-ip>/${ingressNo
 		>fabric-operator/values1.yaml
 
 # step 8. install fabric operator
-kubectl create namespace baas-system
-helm -nbaas-system install fabric -f fabric-operator/values1.yaml --wait fabric-operator
-echo "deploy fabric-operator successfully"
-kubectl get po -nbaas-system
+if [[ ${IGNORE_FABRIC_OPERATOR} != "YES" ]]; then
```

Comment:
can we add a seperate function to do   `baas install` like install_baas\uninstall_baas  ? use this func in the future https://github.com/bestchains/installer/issues/6

Reply:
Will add this function in subsequent pr.